packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3846,6 +3846,13 @@ class Stack extends MultiChildRenderObjectWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=_O0PPD1Xfbk}
 ///
+/// {@tool dartpad}
+/// This example shows a [IndexedStack] widget being used to lay out one card
+/// at a time from a series of cards, each with their respective state.
```

Comment:
Does this sound clearer?
```suggestion
/// at a time from a series of cards, each keeping their respective contents.
```